src/server/cli/test-runner.js

```diff
@@ -37,14 +41,13 @@ async function getTestRunnerOptions(argv = []) {
 		// d2l-test-runner options
 		{
 			name: 'chromium',
-			type: Boolean,
-			description: 'Run tests in Chromium',
+			type: boolStr,
+			description: 'Run tests in Chromium, optionally specifying a version',
```

Comment:
So is the idea with this feature that we might want to run a matrix of browsers + previous versions vs. always running against the latest?

Reply:
That's certainly enabled by this (though I think they'd currently be collapsed to one unless you hacked it by using `--chrome` and `--chromium` etc.) but what really drove it is the number of one-offs I've done where I wanted to go back and see how something rendered in a previous version, or if a bug calls out a specific version.